content/blog/deploying-with-octopus-and-pulumi/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Deploying with Octopus and Pulumi"
+authors: ["sophia-parafina"]
+tags: ["CI"]
+meta_image: "octopus-pulumi.png"
+meta_desc: "Continuous deployment"
```

Comment:
This should be a bit more descriptive. For example, "Continuous Delivery on Octopus Deploy using Pulumi's Step Template" or something. This is the text that is shown to users on Google if this blog post shows up in search results.

Reply:
Shortened to ""Continuous Delivery on Octopus Deploy using Pulumi"

Reply:
Hmm I am not seeing the change to the `meta_desc` here? Perhaps you forgot to push up the commit?